website/docs/d/iam_reference_resource_type.markdown

```diff
@@ -0,0 +1,20 @@
+---
+subcategory : "Account Management"
+---
+
+# ovh_iam_reference_resource_type (Data Source)
+
+Use this data source to list all the IAM resource types.
+
+## Example Usage
+
+```hcl
+data "ovh_iam_reference_resource_type" "types" {
+}
+```
+
+## Argument Reference
+
+## Attributes Reference
+
```

Comment:
Hmm, there is not really a significant ID, this fetches a constant list. So the id is computed as a hash from this list. I don't know what the id's  description should be.

Maybe I am doing something wrong here, since if the list ever changes (ex: new service added), then the ID would change too...

Reply:
Here an example of how an ID of a data source that list registry capabilities is setted:
https://github.com/ovh/terraform-provider-ovh/blob/master/ovh/data_cloud_project_capabilities_containerregistry.go

just a hash generated with the string in input

and here you can find what to say for the ID in the doc:
https://registry.terraform.io/providers/ovh/ovh/latest/docs/data-sources/cloud_project_capabilities_containerregistry


For this kind of ID is not easy and suepr useful but I think we need to documente very fields :)